site2/docs/schema-manage.md

```diff
@@ -0,0 +1,499 @@
+---
+id: schema-manage
+title: Manage schema
+sidebar_label: Manage schema
+---
+
+## Schema AutoUpdate
+
+If a schema passes the schema compatibility check, Pulsar producer automatically updates this schema to the topic it produces by default. 
+
+### AutoUpdate for producer
+
+For a producer, the `AutoUpdate` happens in the following cases:
+
+* If a **topic doesn’t have a schema**, Pulsar registers a schema automatically.
+
+* If a **topic has a schema**:
+
+  * If a **producer doesn’t carry a schema**:
+
+    * If `isSchemaValidationEnforced` or `schemaValidationEnforced` is **disabled** in the namespace to which the topic belongs, the producer is allowed to connect to the topic and produce data. 
+    
+    * If `isSchemaValidationEnforced` or `schemaValidationEnforced` is **enabled** in the namespace to which the topic belongs, the producer is rejected and disconnected.
+
+  * If a **producer carries a schema**:
+  
+    A broker performs the compatibility check based on the configured compatibility check strategy of the namespace to which the topic belongs. 
+    
+    * If it is a new schema and it passes the compatibility check, the broker registers a new schema automatically for the topic.
+
+    * If the schema does not pass the compatibility check, the broker does not register a schema.
+
+![AutoUpdate Producer](assets/schema-autoupdate-producer.png)
+
```

Comment:
@sijie Is this diagram accurate?
If so, I'll add one for `AutoUpdate for consumer`.